symplyphysics/laws/electricity/circuits/resistance_of_the_serial_resistor_coil_capacitor_circuit.py

```diff
@@ -0,0 +1,69 @@
+from sympy import (Eq, solve, sqrt, I)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output,)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+from symplyphysics.definitions import impedance_is_resistance_and_reactance as impedance_law
+from symplyphysics.laws.electricity.circuits import resistivity_of_serial_resistors as serial_law
+
+# Description
+## Consider an electrical circuit consisting of a capacitor, coil, and resistor connected in series.
+## Then you can find the total resistance of such a circuit.
+
+## Law is: R = sqrt(R0^2 + (Xl - Xc)^2), where
+## R - resistance of circuit,
+## R0 - resistance of resistor
+## Xl - inductive reactance,
+## Xc - capacitive reactance.
+
+circuit_resistance = Symbol("circuit_resistance", units.impedance, real=True)
+
+resistance_resistor = Symbol("resistance_resistor", units.impedance, real=True)
+capacitive_reactance = Symbol("capacitive_reactance", units.impedance, real=True)
+inductive_reactance = Symbol("inductive_reactance", units.impedance, real=True)
+
+law = Eq(circuit_resistance, sqrt(resistance_resistor**2 + (inductive_reactance - capacitive_reactance)**2))
+
+# This law might be derived via impedance definition and "resistivity_of_serial_resistors" law.
```

Comment:
Not really. Resistivity of serial resistors is for adding up resistances. Here we should sum impedances. I suggest to add law for sum of impedances.

Reply:
> Not really. Resistivity of serial resistors is for adding up resistances. Here we should sum impedances. I suggest to add law for sum of impedances.

The law for impedances will be literally identical to the law "resistivity_of_serial_resistors".
Should I just copy the law "resistivity_of_serial_resistors" and change its description and name?

Reply:
Yes. But I guess, law for serial resistors is a subset of the law of impedances. So we should add a condition to it. It is valid for non-alternating current, or when there is no reactive load.